transformato/analysis.py

```diff
@@ -264,8 +264,8 @@ def _parse_CHARMM_energy_output(path: str, env: str) -> list:
 
         pot_energies = []
         file_name = ""
-        if env == "waterbox":
-            file_name = f"{path}/ener_solv.log"
+        if env == "waterbox" or env == "complex":
+            file_name = f"{path}/ener_{env}.log"
         elif env == "vacuum":
```

Comment:
I think you could remove the `if ... else` clause and just write out the log file depending on the `env` variable --- that would also include the vacuum log file.